README.md

```diff
@@ -1 +1,90 @@
-# service_camp_capstone
\ No newline at end of file
+# service_camp_capstone
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# portfolio
+
+> I create the nav and headline in my Capstone project, using HTML, CSS(flexbox), Linters.
+> I add the headlines, the projects cards, and also the about page.
+> I created the desktop version using media queries.
+> I Deployed my website using GitHub pages
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML and CSS
+- Media Query
+- Grid Layouts and Flexbox
+- Positioning
+- Technologies: Git and GitHub
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- VS Code - A studio code editor.
+- Node Package Manage: For installing and managing packages used in the project.
+- Linters setup
+- GitHub - For storing our codes and managing the git repo.
+
+### Setup
+
+- Create a Repo
+  - Go to github and create a repo
+  - Copy the Https code to clone
+- Open the Commandline
+  - Navigate To the right directory
+  - Run `Git Clone https://github.com/idungstanley/service_camp_capstone.git`
+  - Run `code .` to open your VScode Editor
+- Create the Github\workflows Folder
+  - Create a Linters.yml file and configure
+- Build and Deploy
+- Capstone project Page: [Click me to view the live demo](https://idungstanley.github.io/service_camp_capstone)
+
+### Install
+
+- install the first linter Webhint
+
+  - Run `npm install --save-dev hint@6.x`
+
+- install Stylelint linter
+
+  - Run `npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x`
+
+- install ESLint linter
+  - Run `npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x`
+
+### Run tests
+
+- run Webhint linter
+
+  - Run `npx hint`. on the root of your directory of your project.
+
+- Run Stylelint linter
+
+  - Run `npx stylelint "**/*.{css,scss}"` on the root of your directory of your project.
+
+- Run ESLint linter
+  - Run `npx eslint`. on the root of your directory of your project.
+
+## Authors
+
+👤 **Sunday Stanley**
+
+- GitHub: [@idungstanley](https://github.com/idungstanley)
+- Twitter: [@idungstanley](https://twitter.com/IdungStanley)
+- LinkedIn: [Sunday (idung) Stanley](https://linkedin.com/in/sundaystanley56)
+
+## Acknowledgments
+
+- Original design idea by <a href="https://www.behance.net/adagio07">Cindy Shin in Behance</a>.
+- Image condent from <a href="https://www.behance.net/adagio07">Virag's Github</a>.
+## 📝 License
+
+This project is [test](./test.md) licensed.
```

Comment:
- [ ] I Think you made a mistake here and wrote test licensed instead of `MID` licensed. Kindly fix the license and update the link, currently it is broken

Reply:
done